project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/.github/workflows/docgen_from_Jira.yaml

```diff
@@ -37,9 +37,11 @@ jobs:
  
       - name: Create commit
         run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
-          git commit -m "Update from JIRA." {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex 
+          git config user.name $GITHUB_ACTOR
+          git config user.email $GITHUB_ACTOR@users.noreply.github.com
+          git add attachments jira_imgs
```

Comment:
Now, is the tex file also being updated? Perhaps you'd want:

```
git add --ignore-missing attachements jira_imgs {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex 
```

I think the `--ignore-missing` might help in cases with one of those files/directories isn't present.

Reply:
--ignore-missing good - the tex file will already exist made by sqrbot-jnr its an update. the other two dirs are always made bu may be empty